core/src/main/scala/org/http4s/rho/bits/FuncParamsMatch.scala

```diff
@@ -0,0 +1,22 @@
+package org.http4s
+package rho.bits
+
+import shapeless.HList
+
+import scala.annotation.implicitNotFound
+
+@implicitNotFound("Expecting a function with parameter types matching (${T}) or (Request[F] :: ${T}). Got (${FU}).")
```

Comment:
```suggestion
@implicitNotFound(
"""Could not bind route to action;
  Expecting a function with parameter types matching:
         (${T}) 
  or     (Request[F] :: ${T})
  found: ${FU}""")
```

Which will yield:
```
[E] [E2] examples/src/main/scala/com/http4s/rho/swagger/demo/MyRoutes.scala:22:22
[E]      Could not bind route to action;
[E]        Expecting a function with parameter types matching:
[E]               (Int :: shapeless.HNil) 
[E]        or     (Request[F] :: Int :: shapeless.HNil)
[E]        found: (Int, Int) => String
[E]      L22:   GET / pathVar[Int] |>> { (i: Int, j: Int) => "Hello World" }
```

I think the extra new lines make it a bit easier to read, and also aligning the types will help somewhat.